src/targeting.js

```diff
@@ -198,6 +199,7 @@ export function newTargeting(auctionManager) {
 
   function getBidsReceived() {
     const bidsReceived = auctionManager.getBidsReceived()
+      .filter(bid => bid.mediaType !== 'banner' || sizeSupported([bid.width, bid.height]))
```

Comment:
What is the reason for this `bid.mediaType !== 'banner'` here ?

Reply:
size mapping only applies to (and filters) `banner` ads currently, so if it is not a `banner` bid then the filter will immediately let it through and not apply any size filtering.